### DISCUSSION TITLE
[@types/d3-graphviz] d3-Graphviz is outdated

### DISCUSSION THREAD

**Discussion by Vithanco** [DefinitelyTyped/DefinitelyTyped#62568] [Issues with a @types package]
D3-Graphviz did move to version 4.4.0 and the type definitions seem not to fit any longer, or have not been incorporating drag & drop. 
I tried to make sense of the existing library, but struggle: https://stackoverflow.com/questions/73831963/what-is-the-right-type-for-graphviz-renderer-instance.

[a comment by an automated account is not shown]

**Comment by DomParfitt** [comment]
@Vithanco are you able to provide a code snippet to show where the types are incomplete/incorrect? To be honest I'm struggling to follow that SO post.

**Reply by Vithanco** [reply]
will do. Give me a day or two, please.

**Comment by Vithanco** [comment]
Dom, thanks for reaching out. 
Currently, I struggle with the setup and cannot reproduce due to setup issues. 
I assume they are on my side, so I will come back, once I got my side in order. 
However, I observe the following message in safari, which doesn't seem to affect anything: 
<img width="678" alt="image" src="https://user-images.githubusercontent.com/4871984/195020148-47f188d9-8b8c-459c-aba2-d57b8ded1eed.png">

**Reply by DomParfitt** [reply]
That's a Javascript runtime error, rather than an error with the type definitions provided here, which would normally suggest that there's a bug of some sort in your code.
